Tests/Unit/psgithub.tests.ps1

```diff
@@ -0,0 +1,69 @@
+#
+# This is a PowerShell Unit Test file.
+# You need a unit test framework such as Pester to run PowerShell Unit tests. 
+# You can download Pester from http://go.microsoft.com/fwlink/?LinkID=534084
+#
+
+$Global:ModuleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
+
+$OldLocation = Get-Location
```

Comment:
I would recommend using `Push-Location` and `Pop-Location` instead of keeping track of a separate variable. In fact, I would avoid using the `Location` commands at all, and instead use fully-qualified paths to everything.